### DISCUSSION TITLE
Hardcoded Active Group Health Metrics

### DISCUSSION THREAD

**Discussion by yorkaspacher** [DiscipleTools/disciple-tools-theme#1594] [Q&A] [answered]
@corsacca Because this is pulling from Group Type, and we have changed Group Type options, so it's not pulling this data
<img width="1350" alt="image" src="https://user-images.githubusercontent.com/47872338/155138726-31b3a514-87c2-4384-8aad-a1610d0b0526.png">

**Comment by corsacca** [comment]
Should all group types be shown in this chart? Team, pre-group, etc?

**Reply by yorkaspacher** [reply]
We have a plugin that replaces the default 4 (pre-group, group, church, team) answers with 2 (small group, body of christ group). So no, we would not want the original values shown, only the options we offer in that field.

**Reply by corsacca** [reply]
That sounds cool for you guys. I'm not sure how this change would affect everyone else who still have team and pre-group as options.
We could have the chart just show all "available" group types. But that could be confusing too.
Also you do have the option of creating your own metric pages too.

**Reply by yorkaspacher** [reply]
Yes, I see… Which of those options would you recommend for cleanest and best longevity with least work and room for confusion?

**Reply by corsacca** [reply]
I'm thinking this chart could show all groups, no matter the type.
And we could brainstorm if another chart that shows more granular detail would be useful.

**Comment by corsacca** [comment, marked as answer]
see https://github.com/DiscipleTools/disciple-tools-theme/issues/1602
